### PR TITLE
1.0: update docs about reset task and relay log purge

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

- update the FAQ about reset task
- clarify the purge-expires config item

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [ ] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

the same purpose as #103

### CODE REVIEW

File: zh/relay-log.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ DM-worker 每次启动时（或在 DM-worker 暂停后 relay log 恢复同步）
 
 ## 数据清理
 
-因为存在文件读写的检测机制，所以 DM-worker 不会清理正在使用的 relay log，也不会清理当前任务稍后会使用到的 relay log。
+因为存在文件读写的检测机制，所以 DM-worker 不会清理正在使用的 relay log，也不会清理当前已有数据迁移任务稍后会使用到的 relay log。
```
Comment (toutdesuite) [inline zh/relay-log.md:91]:
```suggestion
因为存在文件读写的检测机制，所以 DM-worker 不会清理正在使用的 relay log，也不会清理当前已有数据迁移任务之后会使用到的 relay log。
```

File: zh/faq.md [hunk 2]
```diff
@@ -28,28 +28,64 @@ DM 会尝试将包含多个 DDL 变更操作的单条语句拆分成只包含一
...
 ## 如何重置数据同步任务？
 
-在以下情况中，你需要重置整个数据同步任务：
+### relay log 无异常时重置同步任务
 
-- 上游数据库中人为执行了 `RESET MASTER`，造成 relay log 同步出错
+如果数据同步任务所需要的 relay log 不存在异常，可使用如下步骤重置数据同步任务以对数据重新进行迁移：
 
-- relay log 或上游 binlog event 损坏或者丢失
+1. 使用 `stop-task` 停止发生异常的数据迁移任务。
```
Comment (toutdesuite) [inline zh/faq.md:35]:
```suggestion
1. 使用 `stop-task` 停止异常的数据迁移任务。
```

File: zh/faq.md [hunk 3]
```diff
@@ -28,28 +28,64 @@ DM 会尝试将包含多个 DDL 变更操作的单条语句拆分成只包含一
...
 
-此时，relay 处理单元通常会发生错误而退出，且无法优雅地自动恢复，因此需要通过手动方式恢复数据同步：
+2. 清理掉下游已同步的数据。
```
Comment (toutdesuite) [inline zh/faq.md:37]:
```suggestion
2. 清理下游已同步的数据。
```

**Comment by yikeke** [comment]
When you translate this PR, refer to https://github.com/pingcap/docs-dm/pull/180 for translating the same texts. @ireneontheway

**Pull request merged**